docs/rules/use-parameterized-query.md

```diff
@@ -3,7 +3,7 @@ title: Use Parameterized Query
 ruleId: UseParameterizedQuery
 since: 3.16.0
 minJavaVersion: 1.1
-remediationCost: 10
+remediationCost: 20
```

Comment:
What is the reason for this change? The other remediation cost changes provided a corresponding SonarQube reference.